service/raft/raft_group0.cc

```diff
@@ -1029,7 +1029,7 @@ struct sleep_with_exponential_backoff {
     static constexpr std::chrono::seconds _max_retry_period{16};
```

Comment:
Unrelated tip: you can write `constexpr auto _max_retry_period = 16s;` for extra coolness.

Reply:
Only with `using namespace std::chrono_literals` somewhere :(